inc/sail2mcgroup.h

```diff
@@ -99,6 +99,16 @@ typedef enum _sai_l2mc_group_member_attr_t
     SAI_L2MC_GROUP_MEMBER_ATTR_L2MC_OUTPUT_ID,
 
     /**
+     * @brief L2MC Tunnel Endpoint IP.Valid and mandatory if bridge port
+     * is of type SAI_BRIDGE_PORT_TYPE_TUNNEL.
+     *
+     * @type sai_ip_address_t
+     * @flags CREATE_ONLY
+     * @default 0.0.0.0
```

Comment:
The condition is that the new attribute is mandatory and valid if bridge port object id (given in SAI_L2MC_GROUP_MEMBER_ATTR_L2MC_OUTPUT_ID) was created with SAI_BRIDGE_PORT_ATTR_TYPE == SAI_BRIDGE_PORT_TYPE_TUNNEL.
This seems like an indirect condition so not sure on how this can be conveyed using meta data